src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/application/ValidationService.kt

```diff
@@ -6,17 +6,21 @@ import net.leanix.vsm.gitlab.broker.connector.domain.GitLabAssignment
 import net.leanix.vsm.gitlab.broker.shared.exception.AccessLevelValidationFailed
 import net.leanix.vsm.gitlab.broker.shared.exception.InvalidToken
 import net.leanix.vsm.gitlab.broker.shared.exception.OrgNameValidationFailed
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Component
 
 @Component
 class ValidationService(
-    private val gitlabClientProvider: GitlabClientProvider
+    private val gitlabClientProvider: GitlabClientProvider,
+    @Value("\${leanix.gitlab.webhook-url}") private val gitlabWebhookUrl: String,
```

Comment:
Should we make `gitlabWebhookUrl: String?`

Reply:
We don't need it, if you check here if the user doesn't provide the url, we set it as blank:
https://github.com/leanix/vsm-gitlab-broker/blob/d25c925a0b55ce556039c671b83683d44c6a7824/src/main/resources/application.yaml#L18

Reply:
Ah, okay!